utils/wrap_with_fhir_tuple.go

```diff
@@ -0,0 +1,311 @@
+/*
```

Comment:
reviewable code: the generator code

Reply:
Reminds of an undergraduate compiler class project .... (we used CLU, not Go).